File: src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderTrustAccountRepository.java
package uk.gov.justice.hmpps.prison.repository.jpa.repository;

import org.springframework.data.repository.CrudRepository;
import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderTrustAccount;
import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderTrustAccount.Pk;

public interface OffenderTrustAccountRepository extends CrudRepository<OffenderTrustAccount, Pk> {
}

File: src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderTrustAccountRepository.kt
package uk.gov.justice.hmpps.prison.repository.jpa.repository

import org.springframework.data.jpa.repository.JpaRepository
import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderTrustAccount
import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderTrustAccountId

interface OffenderTrustAccountRepository : JpaRepository<OffenderTrustAccount, OffenderTrustAccountId>
